vue-app/src/components/Cart.vue

```diff
@@ -346,6 +352,10 @@ export default class Cart extends Vue {
   }
 
   get filteredCart(): CartItem[] {
+    // In tallying round use committedCart for cart items
+    if (this.$store.state.currentRound.status === RoundStatus.Tallying) {
```

Comment:
I think we'll want this for any round status beyond reallocation. I believe there's a "hasReallocationElapsed" getter or similar.